src/qoi/include/grins/qoi_base.h

```diff
@@ -78,7 +78,9 @@ namespace GRINS
      * Method to allow QoI to cache any system information needed for QoI calculation,
      * for example, solution variable indices.
      */
-    virtual void init( const GetPot& input, const MultiphysicsSystem& system );
+    virtual void init( const GetPot& input,
+                       const MultiphysicsSystem& system,
+                       unsigned int qoi_num );
```

Comment:
Yes, but it's effectively internal --- others subclasses would have to be updated if needed.